brave/brave-core
# [Android] Promotion notification is not cleared when error happens.

Resolves https://github.com/brave/brave-browser/issues/9657.

## Submitter Checklist:

- [x ] Submitted a [ticket](https://github.com/brave/brave-browser/issues) for my issue if one did not already exist.
- [ x] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- Verified that these changes build without errors on
- Verified that these changes pass automated tests (unit, browser, security tests) on

## Test Plan:
1. Open the panel. 
2. Before claiming a promotion, turn off internet.
3. Press "Claim" button.
4. Observe a message "Oops,something is wrong. Please try again later."

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on.
